repls.org

```diff
@@ -939,7 +938,7 @@
    [[https://github.com/cemerick/austin][Austin]] (apparently named after a [[https://en.wikipedia.org/wiki/The_Six_Million_Dollar_Man][sci-fi character from the 70's]]) is an
    alternative to the built-in browser REPL, trying to improve upon it in
    several ways. Since the project came out some other alternatives have come
-   onto the scene, notably [[Weasel][Weasel]] and [[Figwheel][Figwheel]], and unless you have specific
```

Comment:
These are internal links (try them on https://lambdaisland.com/guides/clojure-repls/clojurescript-repls). They link to another section, which then has a link to the project; please do not change them.